modules/installation-generate-aws-user-infra-ignition.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws_user_infra/installing-aws-user-infra.adoc
+
+[id="installation-generate-aws-user-infra-ignition_{context}"]
+= Creating the Kubernetes manifest and Ignition config files
+
+Because you must manually start the cluster machines, you must generate the
+Kubernetes manifest and Ignition config files that the cluster needs to make its
+machines.
+
+[IMPORTANT]
+====
+The Ignition config files that the installation program generates contain
+certificates that expire after 24 hours. You must complete your cluster
+installation and keep the cluster running for 24 hours in a non-degraded state
+to ensure that the first certificate rotation has finished.
+====
+
+.Prerequisites
+
+* Obtain the {product-title} installation program.
+ifdef::restricted[]
+For a restricted network installation, these files are on your bastion host.
```

Comment:
If restricted network is not going to be supported in 4.2, you should consider removing this.

Reply:
@kalexand-rh can confirm for me, but the docs for these restricted network are in for 4.2, so they should be supported. Also, this sentence will only appear in the restricted docs, so if they don't go out for some reason, then this won't appear.